imports/plugins/core/ui/client/components/avatar/avatar.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Avatar from "react-avatar";
+import classnames from "classnames/dedupe";
+import { registerComponent } from "@reactioncommerce/reaction-components";
+
+class ReactionAvatar extends Component {
```

Comment:
Any component that only has a render function should be a simple stateless function.  None of the React lifecycle methods or state handling are being used, so a Component class adds unnecessary overhead. 
This also depends on an email being passed in, so it should have a container wrapping it (currently being done by a Blaze template)